### PR TITLE
Azure readme

### PR DESCRIPTION
**Pull request by assaf13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -382,13 +391,25 @@ If you need more information to decide on your preferred mode, refer to the deta
...
-6. [jq](https://jqlang.github.io/jq/) - Command-line JSON processor.
-7. [curl](https://curl.se/) - Command-line tool for transferring data.
+
+### Azure Prerequisites
+
+1. [establish an Azure App Registration](https://learn.microsoft.com/en-us/azure/healthcare-apis/register-application) and [assign it the necessary role](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=delegate-condition) 
+   for the associated subscription. Note, Assign the Owner role to the app registration on a temporary basis. More specific permissions will be provided later.
+2. The deployment requires access to the Sonar binaries. Establish an Azure Storage account along with a container, and proceed to upload the Sonar binaries to this storage location.
+
+### General Prerequisites
```
Comment (eytannnaim) [inline README.md]: I think we should start with the general prerequisites

File: README.md [hunk 2]
```diff
@@ -382,13 +391,25 @@ If you need more information to decide on your preferred mode, refer to the deta
...
-3. Only if you chose the [CLI Deployment Mode](#cli-deployment-mode), install [Git](https://git-scm.com).
-4. Only if you chose the [CLI Deployment Mode](#cli-deployment-mode), install [Terraform](https://www.terraform.io). It is recommended on MacOS systems to use the "Package Manager" option during installation.
-5. Latest Supported Terraform Version: 1.6.x. Using a higher version may result in unexpected behavior or errors.
-6. [jq](https://jqlang.github.io/jq/) - Command-line JSON processor.
-7. [curl](https://curl.se/) - Command-line tool for transferring data.
+
+### Azure Prerequisites
+
+1. [establish an Azure App Registration](https://learn.microsoft.com/en-us/azure/healthcare-apis/register-application) and [assign it the necessary role](https://learn.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=delegate-condition) 
+   for the associated subscription. Note, Assign the Owner role to the app registration on a temporary basis. More specific permissions will be provided later.
```
Comment (eytannnaim) [inline README.md]: change "later" to "in the future". Later sounds like "down this article"

File: README.md [hunk 3]
```diff
@@ -382,13 +391,25 @@ If you need more information to decide on your preferred mode, refer to the deta
...
+2. The deployment requires access to the Sonar binaries. Establish an Azure Storage account along with a container, and proceed to upload the Sonar binaries to this storage location.
```
Comment (eytannnaim) [inline README.md]: The term blob should also be mentioned, since it where the artifact is stored

File: README.md [hunk 4]
```diff
@@ -382,13 +391,25 @@ If you need more information to decide on your preferred mode, refer to the deta
...
+### General Prerequisites
+
+1. Only if you chose the [CLI Deployment Mode](#cli-deployment-mode), install [Git](https://git-scm.com).
+2. Only if you chose the [CLI Deployment Mode](#cli-deployment-mode), install [Terraform](https://www.terraform.io). It is recommended on MacOS systems to use the "Package Manager" option during installation.
+3. Latest Supported Terraform Version: 1.6.x. Using a higher version may result in unexpected behavior or errors.
+4. [jq](https://jqlang.github.io/jq/) - Command-line JSON processor.
+5. [curl](https://curl.se/) - Command-line tool for transferring data.
+
 
 ## Choosing the Example/Recipe that Fits Your Use Case
```
Comment (eytannnaim) [inline README.md:415]: We should a a column - "cloud provider". Or even better, use a fold-able section per cloud provider examples

File: README.md [hunk 5]
```diff
@@ -835,7 +872,11 @@ The permissions are separated to different policies. Use the relevant policies a
 **NOTE:** When running the deployment with a custom 'deployment_name' variable, you should ensure that the corresponding condition in the AWS permissions of the user who runs the deployment reflects the new custom variable.</br></br>
 **NOTE:** The permissions specified in option 2 are irrelevant for customers who prefer to use their own network objects, such as VPC, NAT Gateway, Internet Gateway, etc.
 
-## IAM Permissions for the DSF Instances
+### IAM Permissions for Azure
+To be able to create Azure resources inside any Azure Account, you need to provide an Azure User or application registration service principal with the required permissions in order to run eDSF Kit Terraform.
+**NOTE:** Assign the Owner role to the user or app registration service principal temporarily. More detailed permissions will be specified at a later stage.
+
```
Comment (eytannnaim) [inline README.md:906]: change to - future release

File: README.md (hunk 4 above)
Comment (assaf13) [reply README.md:415]: I seperared the tables per cloud provider. I don't think we have enough space to add another column

**Pull request merged**